bin/run-samples

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+
+require 'open3'
+require 'thread'
+
+$signals = Queue.new
+
+Signal.trap("HUP") do
+  $signals << true
+end
+
+puts "Parent process #{$$}"
```

Comment:
For the lazy... could we use the english magic globals, there was such a thing wasn't there? I have no idea what this is... I could look it up but... :man_shrugging: